cmd/virtual-workspaces/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	goflag "flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	genericapiserver "k8s.io/apiserver/pkg/server"
+
+	virtualworkspacescmd "github.com/kcp-dev/kcp/pkg/virtual/workspaces/cmd"
+)
+
+// wordSepNormalizeFunc changes all flags that contain "_" separators
+func wordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedName {
+	if strings.Contains(name, "_") {
+		return pflag.NormalizedName(strings.Replace(name, "_", "-", -1))
+	}
+	return pflag.NormalizedName(name)
+}
+
+
+func main() {
+	stopCh := genericapiserver.SetupSignalHandler()
+
+	rand.Seed(time.Now().UTC().UnixNano())
+
+	pflag.CommandLine.SetNormalizeFunc(wordSepNormalizeFunc)
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+
+
+	if len(os.Getenv("GOMAXPROCS")) == 0 {
```

Comment:
Why?

Reply:
Copied from the `openshift-apiserver` command, since the `openshift-apiserver` was used as a basis and model to bootstrap this work.
This might just disappear if we don't need it.

Reply:
also legacy. would remove it.

Reply:
done in https://github.com/davidfestal/kcp/pull/3